data/about-us.js

```diff
@@ -12,6 +22,48 @@ const aboutUsData = {
     title: 'Our story',
     image: '/images/timeline.png',
   },
+  investors: {
+    preTitle: 'Lorem Ipsum', // TODO : Update with text
```

Comment:
Is this wanted?

Reply:
Yes, I added a note on this on the PR body 🙈 

```
Some more notes:

- The Lorem Ipsum text is normal (to be updated later)
```